.github/workflows/compare_translations.py

```diff
@@ -79,10 +82,11 @@ def load_translation(filepath):
     return translation
 
 
-def check_translations():
+
+def check_translations(directory):
```

Comment:
1. Update the docstring to the Google standard 
    1. https://google.github.io/styleguide/pyguide.html

![image](https://github.com/PalisadoesFoundation/talawa/assets/16875803/e48eff96-3bef-4a45-a60d-c2ac118ab076)